### PR TITLE
V1 0 0a

### PR DESCRIPTION
**Pull request by AAndersn**

### CODE REVIEW

**Comment by craig8** [comment]
Are you going to handle the different merge conflicts that are present? @AAndersn

**Comment by AAndersn** [comment]
@craig8  - I think I fixed the conflicts. Please confirm

File: cimgraph/databases/blazegraph/deprecated.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
```
Comment (craig8) [inline cimgraph/databases/blazegraph/deprecated.py:1]: Do we need deprecated.py ?

**Comment by craig8** [comment]
Please merge the PR I made to this branch and then we should remove the test_output folders unless we are going to keep those for "expected output" when we move to automatic testing.

**Review by craig8 — APPROVED** [review]

**Comment by AAndersn** [comment]
Yay!! Thanks so much @craig8 !

**Pull request merged**